langchain4j-parent/pom.xml

```diff
@@ -266,6 +265,13 @@
                 <version>${elastic.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.fasterxml.jackson</groupId>
+                <artifactId>jackson-bom</artifactId>
+                <version>${jackson.version}</version>
+                <type>pom</type>
```

Comment:
Should add `<scope>import</scope>`

Reply:
Oh sorry that comes from #523 which should be merged before